app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,102 +1,86 @@
+// @ts-nocheck
 import React, { useState, useEffect } from "react";
 
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Delete } from "neetoicons";
 import { Button, PageLoader } from "neetoui";
-import { Container, Header, SubHeader } from "neetoui/layouts";
+import { Container, Header } from "neetoui/layouts";
 
-import notesApi from "apis/notes";
+import Item from "components/Common/Item";
 import EmptyState from "components/Common/EmptyState";
+import Menubar from "components/Common/Menubar";
+import EditNotePane from "./Pane/Edit";
 
+import { NOTES_DATA } from "./constants";
 import DeleteAlert from "./DeleteAlert";
 import NewNotePane from "./Pane/Create";
-import Table from "./Table";
 
 const Notes = () => {
-  const [loading, setLoading] = useState(true);
   const [showNewNotePane, setShowNewNotePane] = useState(false);
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
   const [selectedNoteIds, setSelectedNoteIds] = useState([]);
   const [notes, setNotes] = useState([]);
+  const [category, setCategory] = useState("All");
 
   useEffect(() => {
-    fetchNotes();
+    setNotes([...NOTES_DATA]);
```

Comment:
In future, Application might upgrade to user fed values(fetched API data), In that case if any changes are made in [...NOTES_DATA] then it needs to be rendered.

Reply:
> Why do you need a `useEffect` for this? Given that you are using a constant value, `couldn't you pass in that value as the default value of the state during declaration?`

Yes, that too can be done for the purpose of this application, will incorporate the changes.

Reply:
At the moment, useEffect is not needed here. We don't prematurely add code in the mindset of `xyz will happen in the future`. Currently the data is CONSTANT in nature. Whether it would change or not is yet to be seen. PS: There are exceptional cases where have to write code that can incorporate future needs. But here it's not needed. 

A really good read regarding this specific usage of `useEffect` and why it's not required in certain cases:
https://beta-reactjs-org-git-you-might-not-fbopensource.vercel.app/learn/you-might-not-need-an-effect#resetting-all-state-when-a-prop-changes

^ PS: This full article is worth reading.